OxfordBodleian/Juel-Jensen/BDLaethg30.xml

```diff
@@ -36,11 +40,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     
                     <msContents>
                         <summary/>
-                        
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
-                        
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r" to="10v"/>
+                            <title>Unidentified text</title>
+                            <note>The text is for the most part illegible due to faded ink.</note>
```

Comment:
I think that on f. 3r I can see ጸሎት፡ በእንተ፡ ልሳነ፡ ሰብ[እ], but I cannot identify the rest of the legible text with any of the texts I found on BM under that name. But maybe it might still be given in the description as an indication of content.